frontend/src/book-read/store/apis/review.js

```diff
@@ -0,0 +1,73 @@
+import { createApi } from "@reduxjs/toolkit/query/react";
+import { baseQueryWithReauth } from '../../../auth';
+
+
+export const publishReviewApi = createApi({
+  reducerPath: "publishReviewApi",
+  baseQuery: baseQueryWithReauth,
+  endpoints(builder) {
+    return {
+      publishReview: builder.mutation({
+        invalidatesTags: ['Review'],
+        query: (review) => {
+          return {
+            url: `/users/current/books/${review.bookId}/reviews`,
+            body: {
+              score: review.score,
+              title: review.title,
+              text: review.text,
+            },
+            method: "POST",
+          };
+        },
+      }),
+      updateReview: builder.mutation({
+        invalidatesTags: ['Review'],
+        query: (review) => {
+          return {
+            url: `/users/current/books/${review.bookId}/reviews`,
+            body: {
+              score: review.score,
+              title: review.title,
+              text: review.text,
+            },
+            method: "PUT",
+          };
+        },
+      }),
+      getReviewByBook: builder.query({
```

Comment:
The name of this endpoint should start with "fetch" as well to keep consistency.